test/testcafe/spec/NumberChallengePushView_spec.js

```diff
@@ -87,10 +88,10 @@ test
     await checkA11y(t);
     // wait and see that there is only one call for poll
     await t.wait(5000);
-    let warningBox = challengeOktaVerifyPushPageObject.getWarningBox();
     await t.expect(logger.count(() => true)).eql(1);
     await challengeOktaVerifyPushPageObject.waitForErrorBox();
-    await t.expect(warningBox.visible).notOk();
-    const error = challengeOktaVerifyPushPageObject.getErrorBox().textContent;
+    // Check that the reminder prompt is not displayed
+    await t.expect(challengeOktaVerifyPushPageObject.form.getErrorBoxCount()).eql(1);
```

Comment:
Should this be `*.form.getAlertBoxCount` to make more sense? Since the reminder prompt isnt an error but both are considered alerts?